src/App/VPSScan/Scan/RunIPR.hs

```diff
@@ -14,13 +14,14 @@ import qualified Data.HashMap.Strict as HM
 import qualified Data.Vector as V
 import Effect.Exec
 import Data.Aeson
+import Data.Text
```

Comment:
We should do `import qualified Data.Text as T` here and then `T.pack` further down in the file